README.md

```diff
@@ -0,0 +1,106 @@
+OpenIddict<br><sub>The OpenID Connect server you'll be addict to.</sub>
+=======
+
+AppVeyor: [![Build status](https://ci.appveyor.com/api/projects/status/d0d8git3o6lqkvbm?svg=true)](https://ci.appveyor.com/project/aspnet-contrib/core)
+
+Travis: [![Build status](https://travis-ci.org/openiddict/core.svg)](https://travis-ci.org/openiddict/core)
+
+### What's OpenIddict ?
+
+OpenIddict aims at providing a simple and easy out of the box solution 
+to implement an OpenID Connect server for ASP.NET 5.
+
+
+### Why an OpenID Connect Server?
+
+With an OpenID Connect Server you can manage all your users using local 
+password or an external identity provider management for all your applications 
+in one central place, with the power to control who can access your API and 
+the information that is exposed to each client. 
+
```

Comment:
The native UI needs a large rework (it's the "basic" UI I used for ASOS samples):

![image](https://cloud.githubusercontent.com/assets/6998306/10987782/d4fc387c-8436-11e5-8655-1a06d298b121.png)